Frends.Kafka.Produce/Frends.Kafka.Produce/Definitions/SchemaRegistry.cs

```diff
@@ -0,0 +1,221 @@
+
+    /// <summary>
+    /// A comma-separated list of URLs for schema registry instances that are used to register or lookup schemas.
+    /// </summary>
+    /// <example>http://localhost:8081</example>
+    [UIHint(nameof(UseSchemaRegistry), "", true)]
+    public string SchemaRegistryUrl { get; set; }
+
+    /// <summary>
+    /// Authentication credentials source.
+    /// UserInfo = Credentials are specified via the BasicAuthUserInfo property in the form username:password. If BasicAuthUserInfo is not set, authentication is disabled.
+    /// SaslInherit = Credentials are specified via the `Sasl.SaslUsername` and `Sasl.SaslPassword` parameters.
+    /// </summary>
+    /// <example>AuthCredentialsSource.UserInfo</example>
+    [UIHint(nameof(UseSchemaRegistry), "", true)]
+    [DefaultValue(AuthCredentialsSources.UserInfo)]
+    public AuthCredentialsSources BasicAuthCredentialsSource { get; set; }
+
+    /// <summary>
+    /// Basic auth credentials in the form {username}:{password}.
+    /// </summary>
+    /// <example>foo:bar</example>
+    [UIHint(nameof(UseSchemaRegistry), "", true)]
+    public string BasicAuthUserInfo { get; set; }
+
+    /// <summary>
+    /// Enable SSL verification. 
+    /// Disabling SSL verification is insecure and should only be done for reasons of convenience in test/dev environments.
+    /// </summary>
+    /// <example>true</example>
+    [UIHint(nameof(UseSchemaRegistry), "", true)]
+    [DefaultValue(true)]
+    public bool EnableSslCertificateVerification { get; set; }
+
+    /// <summary>
+    /// Specifies the maximum number of schemas CachedSchemaRegistryClient should cache locally.
+    /// </summary>
+    /// <example>1000</example>
+    [UIHint(nameof(UseSchemaRegistry), "", true)]
+    [DefaultValue(1000)]
+    public int MaxCachedSchemas { get; set; }
+
+    /// <summary>
+    /// Specifies the timeout for requests to Confluent Schema Registry.
+    /// </summary>
+    /// <example>30000</example>
+    [UIHint(nameof(UseSchemaRegistry), "", true)]
+    [DefaultValue(30000)]
+    public int RequestTimeoutMs { get; set; }
+
+    /// <summary>
+    /// File path to CA certificate(s) for verifying the Schema Registry's key. 
+    /// System CA certs will be used if not specified.
+    /// </summary>
+    /// <example></example>
+    [UIHint(nameof(UseSchemaRegistry), "", true)]
+    public string SslCaLocation { get; set; }
+
+    /// <summary>
+    /// SSL keystore (PKCS#12) location.
+    /// </summary>
+    /// <example>/path/to/your/keystore.p12</example>   
+    [UIHint(nameof(UseSchemaRegistry), "", true)]
+    public string SslKeystoreLocation { get; set; }
+
+    /// <summary>
+    /// SSL keystore (PKCS#12) password.
+    /// </summary>
+    /// <example>foo</example>
+    [UIHint(nameof(UseSchemaRegistry), "", true)]
+    public string SslKeystorePassword { get; set; }
+
+    /// <summary>
+    /// Schema JSON.
+    /// </summary>
+    /// <example>
+    /// {
```

Comment:
Let's keep examples brief. This is quite a big example value :) Maybe just a couple of fields?

Reply:
Unless of course there is a good reason for such a big example

Reply:
Idea of this big example was to show fields that are tested/supported.

Reply:
Sure, agreed